src/test/resources/features/role/RoleCreate.feature

```diff
@@ -0,0 +1,10 @@
+# feature to review behaviour of create role
+Feature: RoleCreate
+  Scenario: Roles - Registrar datos de un nuevo rol
+    Given 'SSI-A' home page is loaded
+    And Clicking on Personnel menu on 'Header' page
```

Comment:
Please use the format that we mentioned on class.
Example
And click 'Persona' menu on 'Header' page